packages/desktop/views/dashboard/wallet/views/implicit-account-creation-multistep/AccountCreationView.svelte

```diff
@@ -30,8 +40,8 @@
     }
 </script>
 
-<step-content class="flex flex-col items-center justify-between h-full pt-28 w-full">
-    <div class="flex flex-col text-center justify-center px-4 space-y-9 max-w-md w-full">
+<step-content class="flex flex-col items-center justify-between h-full pt-28">
```

Comment:
needs a little love, missing space between items :pray: 
![Screenshot from 2024-02-19 09-19-02](https://github.com/iotaledger/firefly/assets/72078237/393b852f-f8bf-4b4c-80b3-3953cc8d3195)

Reply:
thanks for pushing the changes 🙏🏼